components/table/table-wrapper.js

```diff
@@ -262,6 +267,9 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 				--d2l-table-cell-padding-alt: calc(0.6rem - 1px) 0.6rem 0.6rem 0.6rem;
 				--d2l-table-border-color: var(--d2l-color-gypsum);
 				--d2l-table-header-background-color: #ffffff;
+				--d2l-sortable-button-border-radius: 0.2rem;
+				--d2l-sortable-button-height: 90%;
+				--d2l-sortable-button-focus-height: 80%;
```

Comment:
In the spec, light mode takes up the full height (goes down to the border) when focused, so this might need some tweaking.